exporter/googlecloudexporter/README.md

```diff
@@ -235,6 +235,30 @@ following proxy environment variables:
 If set at Collector start time then exporters, regardless of protocol,
 will or will not proxy traffic as defined by these environment variables.
 
+### Monitored Resources
+
+For metrics and logs, this exporter maps the OpenTelemetry Resource to a Google
+Cloud [Logging\(https://cloud.google.com/logging/docs/api/v2/resource-list) or
+[Monitoring](https://cloud.google.com/monitoring/api/resources) Monitored Resource.
+
+The complete mapping logic can be found [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/internal/resourcemapping/resourcemapping.go).
+That may be the most helpful reference if you want to map to a specific monitored
+resource.
+
+If running on GCP, using the GCP resource detector, as shown above, will populate
+the resource attributes required to map to the appropriate monitored resource.
+
+**Note:** If you are not running on GCP, you still need to choose a [GCP zone or
```

Comment:
Maybe it would make sense to put these sections under another set of headers? One for running on GCP, the other for not running on GCP? Just thinking of ways to try to make it more obvious.

Reply:
done